plugin/pkg/admission/alwayspullimages/admission.go

```diff
@@ -65,6 +66,27 @@ func (a *AlwaysPullImages) Admit(ctx context.Context, attributes admission.Attri
 	if !ok {
 		return apierrors.NewBadRequest("Resource was marked with kind Pod but was unable to be converted")
 	}
+	oldPod, ok := attributes.GetOldObject().(*api.Pod)
+	if !ok {
+		return apierrors.NewBadRequest("Resource was marked with kind Pod but old object was unable to be converted")
+	}
+
+	if attributes.GetOperation() == admission.Update {
```

Comment:
Can we add test cases to cover this change?

Reply:
Try add some

Reply:
added